astra/src/main/java/com/slack/astra/chunk/ReadOnlyChunkImpl.java

```diff
@@ -305,7 +305,7 @@ public void downloadChunkData() {
       // disregarding any errors
       setAssignmentState(
           getCacheNodeAssignment(), Metadata.CacheNodeAssignment.CacheNodeAssignmentState.EVICT);
-      LOG.error("Error handling chunk assignment", e);
+      LOG.error("Error handling chunk assignment for {}", getCacheNodeAssignment().toString(), e);
```

Comment:
nit: the toString() is implied, so you can drop it.
```suggestion
      LOG.error("Error handling chunk assignment for {}", getCacheNodeAssignment(), e);
```